file.c

```diff
@@ -7054,10 +7054,10 @@ const char ruby_null_device[] =
  *  When mode is an integer it must be one or more of the following constants,
  *  which may be combined by the bitwise OR operator <tt>|</tt>:
  *
- *  - +File::RDONLY+: Open for reading only.
- *  - +File::WRONLY+: Open for writing only.
- *  - +File::RDWR+: Open for reading and writing.
- *  - +File::APPEND+: Open for appending only.
+ *  - +File::RDONLY: Open for reading only.
+ *  - +File::WRONLY: Open for writing only.
+ *  - +File::RDWR: Open for reading and writing.
+ *  - +File::APPEND: Open for appending only.
```

Comment:
Why removed `+` after words only?

Reply:
Editing error.  (I should have seen these in the diff.)  Now fixed (nine instances).